api/src/test/java/com/github/commonsrdf/api/AbstractRDFTermFactoryTest.java

```diff
@@ -392,9 +363,9 @@ public void invalidLiteralLang() throws Exception {
 
 	@Test(expected = Exception.class)
 	public void invalidTriplePredicate() {
-		BlankNode subject = factory.createBlankNode("b1");
-		BlankNode predicate = factory.createBlankNode("b2");
-		BlankNode object = factory.createBlankNode("b3");
+		BlankNode subject = factory.createBlankNode();
+		BlankNode predicate = factory.createBlankNode();
+		BlankNode object = factory.createBlankNode();
 		factory.createTriple(subject, (IRI) predicate, object);
```

Comment:
Is this just testing for a ClassCastException? The use of expected=Exception.class is not recommended for that reason that you are not sure where the exception comes from (and using expected=SpecificException.class you are still not sure where the exception comes from)